dockerfiles/Dockerfile.android-ci-base

```diff
@@ -14,14 +9,6 @@ COPY gradle/ /app/gradle/
 ENV GRADLE_OPTS="-Dorg.gradle.daemon=false"
 RUN ./gradlew
 
-# Setup signing
```

Comment:
Signing now takes place in the `android-common` image as this improves caching of the generated key